src/status_im/contexts/communities/overview/view.cljs

```diff
@@ -354,16 +516,12 @@
                           (i18n/label :t/failed-to-fetch-community))
        :container-style {:flex 1 :justify-content :center}}]]))
 
-(defn- community-card-page-view
-  [id]
-  (let [{:keys [joined name images]
-         :as   community} (rf/sub [:communities/community id])]
-    (if community
-      [community-scroll-page id joined name images]
-      [community-fetching-placeholder id])))
-
 (defn view
   [id]
-  (let [id (or id (quo.context/use-screen-params))]
+  (let [community-id (or id (rf/sub [:get-screen-params :community-overview]))
+        community    (rf/sub [:communities/community-overview community-id])
```

Comment:
hi @ulisesmac, Thank you very much for improving the community screen.

While addressing conflicts, please ensure that you are not reverting any changes. Here, we are reintroducing `:get-screen-params` in the community screen.

Reply:
I'll fix it in my next PR, thanks for noticing